components/composed/collection/CollectionList/CollectionList.tsx

```diff
@@ -48,12 +50,16 @@ function CollectionList<T extends OperationType>({
       actions={actions}
       selectable
       data={collections}
+      headerStyle={headerStyle}
+      hideHeader={hideHeader}
     />
   );
 }
 
 interface CollectionListProps {
   data?: CollectionListFragment$key;
+  headerStyle?: "primary" | "secondary";
+  hideHeader?: boolean;
```

Comment:
Let's DRY the list type definitions by extending from PageHeader's types:
```
// Get PageHeader's types:
type HeaderProps = React.ComponentProps<typeof PageHeader>;

...

// Extend CollectionListProps with PageHeader:
// Pick lets us choose which props we want to use
interface CollectionListProps extends Pick<PageHeader, 'headerStyle' | 'hideHeader'> {
    data?: CollectionListFragment$key;
}
```

Now if the definition of `headerStyle` or `hideHeader` changes in PageHeader, we won't have to update the other components

Reply:
@jen-castiron I should do this for each component that uses the props?

Reply:
@1aurend Yep - it's pretty verbose, but it makes sure that the definition isn't repeated outside of `PageHeader`